src/plugins/console/public/application/hooks/use_send_current_request/send_request.ts

```diff
@@ -127,25 +127,18 @@ export function sendRequestToES(args: EsRequestArgs): Promise<ESRequestResult[]>
         }
       } catch (error) {
         let value;
-        let contentType: string | null = '';
+        const { response, body } = error as IHttpFetchError;
+        const contentType = response?.headers.get('Content-Type') ?? 'unknown';
```

Comment:
nit: I'm not sure if we could use the string "unknown" here. Is there a specific reason to not keep an empty string if there is no content-type?